File: app/public_html/script/functions/samples/31.c
int start = 1;
loop(){
    if (getLowHp()){
        float d = getDistToTarget();
        if (!isSlowed() && d > 2)
            charge();
        else if (d <= 1)
            attackMelee();
        else
            moveToTarget();
    }
    else if (start || !isSafeHere()){
        if(moveTo(12.5,12.5))
            start = 0;
    }
    else
        turn(5);
}

File: app/public_html/script/functions/samples/31.py
start = True
def loop():
    global start
    if getLowHp():
        d = getDistToTarget()
        if not isSlowed() and d > 2:
            charge()
        elif d <= 1:
            attackMelee()
        else:
            moveToTarget()
     
    elif start or not isSafeHere():
        if moveTo(12.5,12.5):
            start = False
     
    else:
        turn(5)
 